### DISCUSSION TITLE
New version, some questions

### DISCUSSION THREAD

**Discussion by EMLGVS** [cpvalente/ontime#1061] [Discussion]
I like some upgrades but here some interrogation I have 

-Where can we see a connected clients list?

-Lost the view of what local network was accessible to connect to (previously pink typo on right top side of the screen)

-Why does the editor+cuesheet+operator have a lock icon ? But anyone can modify data / show entry
 
On this, it would be nice to attribute levels of users. Admin vs end users (read-only)

I'm running a show today, I'll post more questions by the day goes.

**Comment by cpvalente** [comment]
Hi @EMLGVS , thank you for this.
I look forward to your questions. For now:

> -Where can we see a connected clients list?

The list of connected clients (as it existed in v2) is in the `Editor` > `Settings` > `Log`
We are working on some improvements for this which will hopefully be in the next release

> -Lost the view of what local network was accessible to connect to (previously pink typo on right top side of the screen)

Same place as the logs, in `Editor` > `Settings` > `Log`

> -Why does the editor+cuesheet+operator have a lock icon ? But anyone can modify data / show entry
On this, it would be nice to attribute levels of users. Admin vs end users (read-only)


The lock indicates that these are what we call "production views", they can be protected by the pin code that is set in `Editor` > `Settings` > `App Settings` > `General Settings` (ouch for this path 😅 ).
If a pin code is defined here a user that is **in a machine that is not where the server is running** will be prompted to enter the pin code

You will find in the feature we have two levels of users `Editor` and `Operator`.
Ontime's philosophy is that there are the following layers of responsibilities
- Production
- Backstage
- Public

Production responsibility means you are able to modify the app data. 
Here we make the distinction between editors and operators (which you see above), where only the editor can currently affect the runtime (manipulate timers and times)

Hope this helps
